sycl/include/CL/sycl/atomic.hpp

```diff
@@ -66,79 +66,79 @@ template <> struct GetSpirvMemoryScope<access::address_space::local_space> {
 };
 
 } // namespace detail
-} // namespace sycl
-} // __SYCL_INLINE_NAMESPACE(cl)
+} // __SYCL_OPEN_NS()
+__SYCL_CLOSE_NS()
 
 #ifndef __SYCL_DEVICE_ONLY__
 // host implementation of SYCL atomics
-__SYCL_INLINE_NAMESPACE(cl) {
-namespace sycl {
+__SYCL_OPEN_NS() {
 namespace detail {
-// Translate cl::sycl::memory_order or __spv::MemorySemanticsMask::Flag
-// into std::memory_order
+// Translate sycl::memory_order or __spv::MemorySemanticsMask::Flag into
+// std::memory_order
 // Only relaxed memory semantics are supported currently
 static inline std::memory_order
 getStdMemoryOrder(__spv::MemorySemanticsMask::Flag) {
   return std::memory_order_relaxed;
 }
 } // namespace detail
-} // namespace sycl
-} // __SYCL_INLINE_NAMESPACE(cl)
+} // __SYCL_OPEN_NS()
+__SYCL_CLOSE_NS()
 
 // std::atomic version of atomic SPIR-V builtins
 
 template <typename T>
 void __spirv_AtomicStore(std::atomic<T> *Ptr, __spv::Scope::Flag,
                          __spv::MemorySemanticsMask::Flag MS, T V) {
-  Ptr->store(V, ::cl::sycl::detail::getStdMemoryOrder(MS));
+  Ptr->store(V, ::__sycl_ns_alias::detail::getStdMemoryOrder(MS));
```

Comment:
```suggestion
  Ptr->store(V, __sycl_ns_alias::detail::getStdMemoryOrder(MS));
```
I think we should use consistent style.